### ISSUE TITLE
1.9.2 client crashes when 1.10.2 server sends a sound to play

### ISSUE THREAD

**Issue by RoboMWM** [ProtocolSupport/ProtocolSupport#256]
Either via a vanilla `/playsound` command or player#playsound. Sounds were custom sounds, did not test with normal sounds.

Wasn't able to get a crash log from the player, but it was consistent/reproducible.

**Comment by RoboMWM** [comment]
And of course it's only after I create a new issue that I see a ticket made before... #223
